lib/x86/cpu_features.c

```diff
@@ -30,6 +30,27 @@
 
 #if HAVE_DYNAMIC_X86_CPU_FEATURES
 
+#ifdef _MSC_VER
+
+static inline void
+cpuid(u32 leaf, u32 subleaf, u32 *a, u32 *b, u32 *c, u32 *d)
+{
+	u32 result[4];
```

Comment:
`int result[4];`, so that the cast below is unnecessary.  Also leave a blank line after the declaration, please